.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+commands:
+  test-nodejs:
+    steps:
+      - run:
+          name: Versions
+          command: npm version
+      - checkout
+      - run:
+          name: Run tests
+          command: npm run ci
+jobs:
+  node-v6:
+    docker:
+      - image: node:6
+        environment:
+          secure: WeX/nE4FbSZkldDQHyHQruDRbJGbCDmerFqVC1Lbx7RYz5pU5pLraFRUWPUcSguq7Pt7iem7iOADtEqoIuurb0tttc91ilVIOTvtICixATFpRsiuilMqEo5YKNKwmsTiJzOnaoMEnxWOZ8WK+mVBN5h1qTGzg1gEIXBU8wjy1eSQ2+GEGC/NvtKjMwn3ew5XXqr5w7sU7AvoVBBJkUBucG09DVxIW9iB+Pz+k1kCcaCWunZ3r1tSOaIZoKCIW6SZCy2XD0ns0GKdBw+AygxVqDN5LpSj9EJ27wQpkpMcdFnpOyUoijHuqa8d8mnofWrkd3UQyIi1bz/Dq2wmSfLHfxb9+U/iWSi9q3ed2YfeX4+Q3of80onucqD1gTzjwZU+KZi9XSlVEgKEv+WwmTPE+okURGsaONCB3QLzgTo27cv/ACe8GU4UhomkrmDES8y6P1Zj49Np9GOOkYMzerUyV++qaTF6ugi8Z1k93/Put7EaESVmZJxxrtuhm70Dq96Eftrag9ypByYEgRYVKRqL7840bBb+I8J5pC8+uS5xBcusI9amYv01qdqP43F3f3XvRI4KHEG/OO4TfTSmt1htiDKKnciC2/6K6766y3wCOzxEeD2XPie4VkGsSTMJegxkPH37imogXJ/p4Ket7pprZXDnkEsiMn1b+CgTSaNb8ms=
```

Comment:
I think this `secure` was used in Travis for Coveralls integration. Do we still need it with CircleCI? Does Coveralls still happen with CircleCI?

Reply:
I don't see it called in Travis either anywhere, I wonder how it used to work. Since CircleCI integration is not enabled, not sure if it works.